src/client/api.cpp

```diff
@@ -275,7 +275,8 @@ PYBIND11_MODULE(pyprt, m) {
 	        .def("get_faces", &GeneratedModel::getFaces, doc::GmGetF)
 	        .def("get_report", &GeneratedModel::getReport, doc::GmGetR)
 	        .def("get_cga_prints", &GeneratedModel::getCGAPrints, doc::GmGetP)
-	        .def("get_cga_errors", &GeneratedModel::getCGAErrors, doc::GmGetE);
+	        .def("get_cga_errors", &GeneratedModel::getCGAErrors, doc::GmGetE)
+			.def("get_attributes_values", &GeneratedModel::getAttributesValues, doc::GmGetAttr);

```

Comment:
consider naming the new function "get_attributes" so it is symmetrical with the existing functions